packages/ui/src/components/va-date-input/VaDateInput.demo.vue

```diff
@@ -225,19 +228,19 @@ export default {
       value: getStaticDate(),
       range: { start: getStaticDate(), end: nextWeek },
       dates: [getStaticDate(), nextWeek],
-      dayView: { type: 'day', month: 3, year: 2013 },
+      dayView: { type: 'day', month: 3, year: 2013 } as DatePickerView,
       string: getStaticDate().toString(),
-      strings: [getStaticDate().getTime() + 1e9, getStaticDate().toString()],
-      stringRange: { start: getStaticDate().toString(), end: getStaticDate().getTime() + 1e9 },
+      strings: [getStaticDate().getTime() + 1e9, getStaticDate().toString()] as DateInputModelValue,
+      stringRange: { start: getStaticDate().toString(), end: getStaticDate().getTime() + 1e9 } as unknown as DateInputModelValue,
```

Comment:
```suggestion
      stringRange: { start: getStaticDate().toString(), end: getStaticDate().getTime() + 1e9 } as DateInputModelValue,
```

This must work. Let's change DateInput type to

```ts
string | number | Date
```